### PR TITLE
RAT-450: Harmonize log messages to ease migration for RAT consumers of v0.17

### PR DESCRIPTION
**Pull request by ottlinger**
Thanks for pointing me to the deprecation messages and kudos for implementing such a cool mechanism that helps all users of RAT.

### CODE REVIEW

**Comment by ottlinger** [comment]
@Claudenw feel free to merge if changes are ok for you - thanks.

**Comment by Claudenw** [comment]
This change is dependant upon RAT-399 and should wait until after that is merged, or this code could be merged in to RAT-399 first.

**Review by Claudenw — APPROVED** [review]
LGTM

**Pull request merged**
